components/service/profile/ProfileModal/JobSelectbar.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import React, { useState } from 'react';
+
+type Job = (typeof jobs)[number];
+
+const jobs = [
+  '사무직',
+  '연구·개발직',
+  '서비스직',
+  '생산직',
+  '공무원',
+  '프리랜서',
+  '자영업자',
+  '군인',
+  '취업 준비생',
+  '대학생',
+  '고등학생 이하',
+  '무직',
+  '기타',
+] as const;
+
+export default function Selectbar() {
```

Comment:
@Kristyn-Yoon 
```suggestion
export default function JobSelectbar() {
```
여기도 실제 컴포넌트 이름과 맞춰주세요!! Selectbar 보단 라벨이 직업에 한정되어 있으니 파일이름과 동일하게 JobSelectbar가 나을거 같습니다

Reply:
반영하였습니다. 감사합니다!